test/server/dispatcher.test.ts

```diff
@@ -818,3 +818,51 @@ describe("given a request that contains the OpenApi basePath", () => {
     expect(response.body).toBe("ok");
   });
 });
+
+describe("given a request that contains the differently cased path", () => {
```

Comment:
These new tests would pass in the existing codebase, I believe. I added them for clarity of purpose, but because Registry is already case-insensitive the tests pass even though Dispatcher can't find the operation without the new changes.